packages/PostRector/Application/PostFileProcessor.php

```diff
@@ -31,11 +31,11 @@ public function __construct(
     ) {
         $this->postRectors = [
             // priority: 650
-            $classRenamingPostRector,
-            // priority: 600
             $nameImportingPostRector,
-            // priority: 500
+            // priority: 600
             $useAddingPostRector,
+            // priority: 500
+            $classRenamingPostRector,
```

Comment:
the re-index to set name importing and use adding post rector first is on purpose to know that auto import collecting the statements that need to be added, which class renaming depends.